Arrowgene.Ddon.GameServer/Handler/LobbyLobbyDataMsgHandler.cs

```diff
@@ -35,12 +35,9 @@ public override void Handle(GameClient client, StructurePacket<C2SLobbyLobbyData
             res.RpcPacket = packet.Structure.RpcPacket;
             res.OnlineStatus = client.Character.OnlineStatus;
 
-            if(!LobbyStageIds.Contains(client.Character.Stage.Id))
+            if(!LobbyStageIds.Contains(client.Character.Stage.Id) && client.Party != null)
             {
-                if(client.Party != null)
-                {
-                    client.Party.SendToAllExcept(res, client);
-                }
+                client.Party.SendToAllExcept(res, client);
```

Comment:
Instead of this, move the party check outside the if else block, as now both conditions use client.Party. Oversight on my part too lol

Reply:
Done.